src/components/Home/Home.tsx

```diff
@@ -8,16 +8,17 @@ import type { DeviceType } from "@/types";
 
 export type Props = {
   devices: DeviceType[];
+  onNavigateToDevice: (deviceKey: DeviceType["key"]) => void;
 };
```

Comment:
Is there a reason we export these `Props`. I can't see them been useful elsewhere and if they are, it would be better if had a more accurate name like `HomeProps`

Reply:
The reason is for use in tests

Reply:
Can you expand please? Tests should only test the I/O and not internals. How does exporting this type help tests?

Reply:
as you said earlier - it should be possible to do testing whilst being typesafe

<img width="812" alt="Screenshot 2024-07-22 at 14 13 53" src="https://github.com/user-attachments/assets/aa054d27-a105-4f11-9cd1-8b286cfc97a7">

here is an example of how I use types:

<img width="1073" alt="Screenshot 2024-07-22 at 14 16 47" src="https://github.com/user-attachments/assets/501c130e-4eef-432b-8aff-0e7b12b7b250">